run_race.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+import queue
+import runner_class
+import os
+
+
+class RunRace:
+    def __init__(self, runners: set, race_length: int):
+        self.runners = runners
+        self.race_length = race_length
+        self.places = queue.Queue(len(runners))
+
+    def start_race(self):
+        loop = asyncio.get_event_loop()
+        for current_runner in self.runners:
+            loop.create_task(
+                current_runner.run(self.race_length - current_runner.start_point, self.__print_current_stat,
+                                     self.__register_place_of_runner))
+        pending = asyncio.Task.all_tasks()
+        results = loop.run_until_complete(asyncio.gather(*pending))
+        print("the winner is {}".format(self.places.queue.popleft().name))
```

Comment:
In python, instead of using Stacks/Queues the convention is use list

You can do with python list everything that is done with Stacks/Queues and more (and with sometimes even with better time complexity)